app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/logItemDetails/logItemDetails.jsx

```diff
@@ -45,6 +45,10 @@ const messages = defineMessages({
     id: 'TestItemDetailsModal.parametersLabel',
     defaultMessage: 'Parameters:',
   },
+  testCaseId: {
+    id: 'TestItemDetailsModal.testCaseId',
+    defaultMessage: 'Test case id:',
```

Comment:
Please add corresponding translations for this message (as for testCaseId in `src/pages/inside/stepPage/modals/testItemDetailsModal/testItemDetailsModal.jsx`)

Reply:
It already exist, bcs we are using same id `TestItemDetailsModal`

Reply:
Sorry, I missed it.